app/views/collaborations/index.html.erb

```diff
@@ -85,7 +84,7 @@ HEREDOC
         <div class="collaboration <%= collaboration.style_class %> collaboration_<%= collaboration.id %>" data-id="<%= collaboration.id %>">
           <h3><%= collaboration.title %></h3>
           <div style="margin-bottom: 5px;" class="description">
-            <a href="<%= oauth_url(:service => :google_drive, :return_to => (request.url)) %>"><%= t 'google.drive.upgrade.description', "To access this collaboration you must authorize Canvas to access your Google Drive account" %></a>
+            <a href="<%= oauth_url(:service => :google_drive, :return_to => (request.url)) %>"><%= t 'google.drive.upgrade.description', "To access this Collaboration you must authorize Canvas to access your Google Drive account" %></a>
```

Comment:
-1 ... For a heading or button, it's appropriate to use title case and capitalize nouns/adjectives. However for a paragraph, we should stick with sentence case and only capitalize proper nouns (or the first word).